src/gis-utils/components/ol-geolocate.ts

```diff
@@ -80,4 +85,26 @@ export class Geolocate extends ol.control.Control {
     );
     return feature;
   }
+
+  private _addPositionFeature(pos, view, source, positionFeature) {
+    console.debug('ol-geolocate::_addPositionFeature', pos.coords);
+    this.addPositionFeatureCounter++;
+    const zoomLevel = this.options.zoomLevel ? this.options.zoomLevel : 12;
+    const coordinates = ol.proj.transform(
+      [pos.coords.longitude, pos.coords.latitude],
+      'EPSG:4326',
+      view.getProjection()
+    )
+
+    if (this.addPositionFeatureCounter < 3) {
+      view.setCenter(coordinates);
+      view.setZoom(zoomLevel);
+      positionFeature.setGeometry(coordinates ? new ol.geom.Point(coordinates) : null);
+    }
+    positionFeature.setGeometry(coordinates ? new ol.geom.Point(coordinates) : null);
```

Comment:
deze zal 2x uitgevoerd worden ils de if true is, nee? best die lijn gewoon uit de if halen?